vendor/k8s.io/kubernetes/pkg/kubectl/cmd/rollout/rollout_pause.go

```diff
@@ -119,17 +132,36 @@ func (o *PauseConfig) CompletePause(f *cmdutil.Factory, cmd *cobra.Command, out
 
 func (o PauseConfig) RunPause() error {
```

Comment:
@deads2k @kargakis this and the resume are identical except the success message (paused/resumed) and the mutateFn. You think it is OK to a little copy here or a helper would be better?